website/source/api/auth/approle/index.html.md

```diff
@@ -69,9 +69,17 @@ enabled while creating or updating a role.
 - `role_name` `(string: <required>)` - Name of the AppRole.
 - `bind_secret_id` `(bool: true)` - Require `secret_id` to be presented when
   logging in using this AppRole.
-- `bound_cidr_list` `(array: [])` - Comma-separated string or list of CIDR
+- `bound_cidr_list` `(array: [])` - Deprecated: Comma-separated string or list of CIDR
   blocks; if set, specifies blocks of IP addresses which can perform the login
   operation.
+- `secret_id_bound_cidrs` `(array: [])` - Comma-separated string or list of CIDR
+  blocks; if set, specifies blocks of IP addresses which can perform the login
+  operation.
+- `token_bound_cidrs` `(array: [])` - Comma-separated string or list of CIDR
```

Comment:
We might want to keep `token_bound_cidrs` to behave as just that and not mix it up with `secret_id_bound_cidrs`. It is less confusing that way and becomes intuitive by their names.

Reply:
Yes, I thought of that approach too. I'm happy to go either way. The reason I didn't pick it initially is because, then the user would have to specify bound CIDRs in both parameters. I thought that could be clunky to use. Also, I _think_ they'd provide identical CIDRs in each list, and that it would work fine even if they didn't, but it would be worthwhile to carefully think through scenarios and perhaps add tests.

On the other hand, in the current scenario they could already provide differing CIDR blocks in each parameter. :-)

Reply:
I'm much more happy with users specifying the same CIDRs in both parameters than with the secret ID value doing double-duty. Also, in many cases where users might have put the same exact values, users will only need to specify token cidrs; secret id cidrs end up simply preventing a token that can't be used from being created and taking up a lease.

Reply:
Cool cool. Will update.

Reply:
@jefferai have pushed commits incorporating all feedback so far.

Reply:
I still see all three values being set via bound_cidr_list...